build.rs

```diff
@@ -126,14 +126,18 @@ fn build_nasm_files() {
     config_include_arg.push_str(&out_dir);
     config_include_arg.push('/');
     let mut nasm = nasm_rs::Build::new();
+    nasm.min_version(2,14,0);
```

Comment:
`cargo fmt` is failing here due to spacing.
```suggestion
    nasm.min_version(2, 14, 0);
```